app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.players.player import Player
```

Comment:
It is also a good practice to leave a blank line between different types of import